scripts/link_monorepo_to_plugin.sh

```diff
@@ -46,13 +46,13 @@ yarn link-all
 cd ..
 
 # Get all the packages inside the yoastseo package.json that are scoped using @yoast
-packages=($(cat package.json | tr '"' '\n' | grep -w @yoast))
+packages=($(ls javascript/packages))
 
 for package in ${packages[@]}
 do
-   echo -e "Linking ${package}\n"
+   echo -e "Linking @yoast/${package}\n"
```

Comment:
This will also attempt to link `@yoast/yoast-components` and `@yoast/yoastseo` and fail (without existing). Is this by design?

Reply:
Yes, I didn't care about this error because it doesn't break.